test/fulfillment_request_test.rb

```diff
@@ -1,21 +1,21 @@
 require 'test_helper'
 
-class FulFillmentTest < Test::Unit::TestCase
+class FulFillmentRequestTest < Test::Unit::TestCase
   def setup
     fake "orders/450789469/fulfillment_requests/695890229", :method => :get, :body => load_fixture('fulfillment_request')
   end
 
-  context "#cancel" do
-    should "be able to cancel fulfillment request" do
+  context "#mark_as_failed" do
+    should "be able to mark_as_failed a fulfillment request" do
       fulfillment_request = ShopifyAPI::FulfillmentRequest.find(255858046, :params => {:order_id => 450789469})
 
       cancelled = ActiveSupport::JSON.decode(load_fixture('fulfillment_request'))
-      cancelled['failure_message'] = 'cancelled'
-      fake "orders/450789469/fulfillments/695890229/cancel", :method => :post, :body => ActiveSupport::JSON.encode(cancelled)
+      cancelled['failure_message'] = 'failure reason'
+      fake "orders/450789469/fulfillment_requests/695890229/mark_as_failed", :method => :post, :body => ActiveSupport::JSON.encode(cancelled)
```

Comment:
Since you changed the method from `post` to `put` above, do you have to change the method here too?